druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/JDBCExtractionNamespace.java

```diff
@@ -123,6 +123,11 @@ public boolean isCacheEnabled() {
         return cacheEnabled;
     }
 
+    @Override
+    public boolean isFlatBufferNamespace() {
```

Comment:
Instead of adding this method in every namespace, either:
1. Add default implementation in ExtractionNamespace interface with false
2. Better - create a method 'getNamespaceType' in ExtractionNamespace which returns appropriately for each namespace ("jdbc", "flatbuffer" etc)